### PR TITLE
Records Editor integration, closes #13

### LINKED ISSUE #13: Integrate the record editor

### PR DESCRIPTION
**Pull request by topless**
Integration of the invenio-records-editor module.

### CODE REVIEW

**Review by ntarocco — APPROVED** [review]

**Review by ntarocco — CHANGES_REQUESTED** [review]
You are missing the dependency `invenio-records-editor`, and maybe add it also the requirements file.

File: invenio_app_ils/ext.py [hunk 1]
```diff
@@ -24,6 +27,8 @@ def __init__(self, app=None):
     def _init_app(self, app):
         """Flask application initialization."""
         self._init_config(app)
+        _editor_blueprint = create_editor_blueprint(app)
```
Comment (ntarocco) [inline invenio_app_ils/ext.py]: Can you please remove this due to: https://github.com/inveniosoftware/invenio-records-editor/pull/5

File: setup.py [hunk 2]
```diff
@@ -40,10 +40,11 @@
     "Flask-BabelEx>=0.9.3",
     "Flask-Debugtoolbar>=0.10.1",
     "invenio[postgresql,elasticsearch6,base,auth,metadata]~=3.0.0",
-    "invenio-app>=1.0.3",
+    "invenio-app>=1.0.4",
     "invenio-config>=1.0.1",
     "invenio-circulation==0.1.0.dev20180000",  # TODO: add a version here
     # when released, need for Travis and isort
+    "invenio-records-editor==1.0.0a1"
```
Comment (ntarocco) [inline setup.py]: Can you please `==1.0.0a2`

**Review by ntarocco — COMMENTED** [review]
Can you please squash all commits to keep only 1 and remove the merge commits?
We can always squash when merging eventually.

**Comment by topless** [comment]
Mr. polite @ntarocco I think I covered everything, please be kind enough to let me know if there is anything else I can help you with. I wish you a great day and even greater weekend. May the horse be with you.

**Review by ntarocco — APPROVED** [review]

**Pull request merged**